docs/document/website/static/css/custom-kuc.css

```diff
@@ -80,3 +80,9 @@ table th[class$="__table__header__cell"] {
 #attachment div[class$="__group"] {
   display: block;
 }
+li+li {
+  margin-top: 0;
+}
```

Comment:
I thought it's not good to pollute global CSS.
If this is for Tabs component, you should start with #tabs.